pkg/telemetry/provider.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
-	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp"
```

Comment:
why are we switching to HTTP? gRPC is supposed to be more efficient and it's the native alternative in Go.

Reply:
I agree on using a port different from 443 😄. Why not use the default 4317? I'd argue that is even less surprising, since it's makes clear what is being sent is OTLP telemetry data.

I'm not sure about the simplicity argument. It will be piri instances talking to the server, so I don't think there is a need to optimize for compatibility (at least for now). Also, given the nature of the data I'm not sure we will be using curl to troubleshoot either.

Of course we can add it later, but it looks like it is actually less (or same) effort to use gRPC from the get go. Also, deprecating something (OTLP over HTTP in this case) later when a lot of servers are using it will be difficult.